fileio/ft_read_data.m

```diff
@@ -1026,32 +1039,11 @@
     % recordings. The code currently concatenates these trials.
     % We could set this up as separate "trials" later.
     % We could probably add "boundary events" in EEGLAB later
-    
```

Comment:
It is not clear to me why this recent update to the code has been reverted

Reply:
Please see the comment above